internal/xds/translator/testdata/out/xds-ir/http-route-direct-response.routes.yaml

```diff
@@ -24,3 +52,60 @@
         header:
           key: X-Set-Header
           value: set-502
+    - directResponse:
+        body:
```

Comment:
can we add a test case that actual changes the value from 4096 to something higher

Reply:
is this a case with length 4097? which is invalid before this patch.